LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2014-2022 Nimble.
```

Comment:
Prefer to use the latest version of the Git template (https://github.com/nimblehq/git-template/blob/main/LICENSE)

```suggestion
Copyright (c) 2014 and onwards Nimble.
```

Reply:
Thanks for the suggestion and link 🙌 
Updated in 73721668b52c1735d52b3d3f4d11b59a7d5d2b9e